docs/source/deployment/lambda.md

```diff
@@ -58,6 +59,226 @@ exports.graphqlHandler = server.createHandler();
 
 Finally, **pay close attention to the last line**. This creates an export named `graphqlHandler` with a Lambda function handler.
 
+## Deploying with the AWS Serverless Application Model (SAM)
+
+The [AWS Serverless Application Model (AWS SAM)](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html) is an open-source framework that you can use to build serverless applications on AWS.
+
+### Configuring your SAM template.yaml
+
+An AWS SAM template file closely follows the format of an AWS CloudFormation template file, which is described in [Template anatomy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html) in the AWS CloudFormation User Guide.
+
+For the sake of this example, the following file can just be copied and pasted into the root of your project.
+
+```
+# template.yaml
+
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  GraphQLFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: graphql/
+      Handler: app.graphqlHandler
+      Timeout: 5 #
+      MemorySize: 256
+      Runtime: nodejs14.x
+      Events:
+        AutomaticPersistedQueries:
+          Type: Api
+          Properties:
+            Path: /
+            Method: get
+        GraphQL:
+          Type: Api
+          Properties:
+            Path: /
+            Method: post
+        ApolloSandbox:
```

Comment:
I'm not sure what this means.  I think people will think that this is what's required to serve the landing page that redirects to Sandbox.  But IIUC this is what is needed in order to talk to your server from any browser context, including your web app.  Just call it CORS or something?